scripts/create-tables.sh

```diff
@@ -15,16 +15,50 @@ bq mk \
   --time_partitioning_type DAY \
   --clustering_fields transactionType \
   ${BQ_TRANSACTIONS_TABLE} \
-  ${SCRIPT_DIR}/../hedera-etl-bigquery/src/main/resources/schema.json
+  ${SCRIPT_DIR}/../hedera-etl-bigquery/src/main/resources/transactions-schema.json
 
 bq mk \
   --table \
   --description "Hedera ETL Errors" \
   ${BQ_ERRORS_TABLE} \
-  ${SCRIPT_DIR}/../hedera-etl-bigquery/src/main/resources/errors_schema.json
+  ${SCRIPT_DIR}/../hedera-etl-bigquery/src/main/resources/errors-schema.json
 
 bq mk \
   --table \
   --description "BigQuery deduplication job state" \
   ${BQ_DEDUPE_STATE_TABLE} \
-  ${SCRIPT_DIR}/../hedera-dedupe-bigquery/state-schema.json
+  ${SCRIPT_DIR}/../hedera-etl-bigquery/src/main/resources/state-schema.json
+
+bq mk \
+  --table \
+  --description "Transaction Types" \
+  ${BQ_TRANSACTION_TYPES_TABLE} \
+  ${SCRIPT_DIR}/../hedera-etl-bigquery/src/main/resources/transaction-types-schema.json
+
+# Not using 'bq load' or 'bq insert' because of lack of flag to specify project_id. That'd require operator to
```

Comment:
`--project_id` is described in the global flags section https://cloud.google.com/bigquery/docs/reference/bq-cli-reference#bq_global_flags. I missed it at first too.

Reply:
ahh. removed the invalid comment.
Leaving query as is, no point spending time to change it.